assets/js/views/tagview.js

```diff
@@ -0,0 +1,27 @@
+var TagView = Backbone.View.extend ({
+  template: AppTemplates.taglist,
+
+  el: '#tags',
+
+  initialize: function() {
+    this.listenTo(this.collection, 'change', this.render);
+    this.render();
+  },
+
+  render: function() {
+    var _this = this;
+
+    this.collection.fetch().then(function() {
+      var html = _this.template(_.filter(_this.collection.models, function(model) {
```

Comment:
Instead of having to do all of this work around to get `_.filter` and `collection.models`: backbone gives you all of the array functions from underscore and makes them available on a collection: http://backbonejs.org/#Collection-Underscore-Methods